### PR TITLE
refactor(dms/rocketmq_user): fix dms rocketmq user lint issues

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix dms rocketmq user lint issues
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix dms rocketmq user lint issues
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDmsRocketMQUser_basic'                 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQUser_basic -timeout 360m -parallel 4 
=== RUN   TestAccDmsRocketMQUser_basic 
=== PAUSE TestAccDmsRocketMQUser_basic
=== CONT  TestAccDmsRocketMQUser_basic
--- PASS: TestAccDmsRocketMQUser_basic (722.84s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       722.889s
```

### CODE REVIEW

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go [hunk 1]
```diff
@@ -172,7 +173,7 @@ func resourceDmsRocketMQUserCreate(ctx context.Context, d *schema.ResourceData,
 	return resourceDmsRocketMQUserRead(ctx, d, meta)
 }
 
-func buildCreateRocketmqUserBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateRocketmqUserBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (Lance52259) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]: If the config is not used, we should remove it from the argument list.

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go [hunk 2]
```diff
@@ -237,7 +238,7 @@ func resourceDmsRocketMQUserUpdate(ctx context.Context, d *schema.ResourceData,
 	return resourceDmsRocketMQUserRead(ctx, d, meta)
 }
 
-func buildUpdateRocketmqUserBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateRocketmqUserBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (Lance52259) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]: ditto.

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go [hunk 3]
```diff
@@ -103,7 +104,7 @@ Value options: **PUB|SUB**, **PUB**, **SUB**, **DENY**.`,
 	}
 }
 
-func DmsRocketMQUserPermsRefSchema() *schema.Resource {
+func RocketMQUserPermsRefSchema() *schema.Resource {
```
Comment (chengxiangdong) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]:
```suggestion
func rocketMQUserPermsRefSchema() *schema.Resource {
```
it seems it should be a private function

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]: done

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]: done

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go]: done

**Pull request merged**